util/microservice-sdk/js/lib/kafka/Producer.js

```diff
@@ -80,20 +80,30 @@ class Producer {
     this.isReady = false;
     this.producer = new Kafka.Producer(this.config.kafka);
 
-
-    // To use the event 'delivery-report', you must set request.required.acks to 1 or -1 in topic
-    // configuration and dr_cb (or dr_msg_cb if you want the report to contain the message payload)
-    // to true in kafka options.
-    if (this.config.kafka.dr_cb || this.config.kafka.dr_msg_cb) {
+    if (this.isDeliveryReportActived()) {
```

Comment:
I think 'Actived' doesn't exist. My suggestion it to change for 'Enabled' or 'Active'.